Products/CMFPlone/skins/plone_form_scripts/discussion_reply.cpy

```diff
@@ -14,6 +14,9 @@ from Products.CMFPlone import PloneMessageFactory as _
 mtool = getToolByName(context, 'portal_membership')
 dtool = getToolByName(context, 'portal_discussion')
 req = context.REQUEST
+if req.get('REQUEST_METHOD', '').upper() != 'POST':
+    from zExceptions import Forbidden
```

Comment:
@mauritsvanrees it is an accepted practice to do the imports in the middle of the code?